products-service/db/createProduct.ts

```diff
@@ -0,0 +1,23 @@
+import {Client} from "pg";
+import {dbOptions} from "./constants";
+import {getAllProductsFromDb} from "./getAllProducts";
+
+
+export const createProductInDb = async (product) => {
+  const client: Client = new Client(dbOptions)
+  await client.connect();
+
+  try {
+    await client.query(`
+      INSERT INTO products (title, description, price) VALUES
+      ('${product.title}', '${product.description}', ${product.price})
```

Comment:
this code is vulnerable for sql injections, try to use Parameterized query. https://node-postgres.com/features/queries